department-of-veterans-affairs/va.gov-cms
# VACMS-000: Update list of content served from Drupal.

## Description

Needs an issue.

## Testing done

Go to https://pr6793-mzapdu3xev7oab2bvpzwxxbubdevkrkr.ci.cms.va.gov/admin/content/audit/drupal-managed-urls 

## Screenshots

<details><summary>Screenshot</summary>
![www_va_gov_pages_served_from_Drupal___VA_gov_CMS](https://user-images.githubusercontent.com/643678/138873443-08433861-bee7-4660-be09-f7eae53cc84e.png)





</details>





## QA steps

This number 

<img width="835" alt="www_va_gov_pages_served_from_Drupal___VA_gov_CMS" src="https://user-images.githubusercontent.com/643678/139363909-ec27289b-52b8-4608-8c60-6c63409f39b9.png">

Should be _fairly_ close to the number in Grafana dashboard

<img width="991" alt="Slack___vfs-platform-builds___Digital_Service___VA___4_new_items" src="https://user-images.githubusercontent.com/643678/139364060-735b70e7-5ba4-4436-8398-afccfa2182aa.png">

We could make this closer, but woudl require a lot more filters to filter out staff profiles and Q&As that do not have standalone URLs. 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
